manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -152,6 +152,26 @@ metadata:
   name: internal-feature-states.csi.vsphere.vmware.com
   namespace: vmware-system-csi
 ---
+apiVersion: v1
```

Comment:
Have we tried considering usage of `InitContainers` section to solve service dependency
Refer: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/

Reply:
No, my goal is to spread Pods on multiple master nodes, so no two pods get scheduled on the single master node.

Reply:
For now, I just want to organize resources as per best practice suggested [here](https://kubernetes.io/docs/concepts/cluster-administration/manage-deployment/#organizing-resource-configurations). Service goes before Deployment. 

I will create separate PR for multi replica support after analyzing the advance scheduling options we are discussing on this PR.

Reply:
Sounds good! Please create an issue to track the same